UnityProject/Assets/Scripts/Objects/Cargo/ConveyorBelts/ConveyorBelt.cs

```diff
@@ -100,15 +100,7 @@ private void Transport(UniversalObjectPhysics item, float ConveyorBeltSpeed)
 			if (item.NewtonianMovement.magnitude > ConveyorBeltSpeed) return;
 			item.Pushing.Clear();
 
-			if (item.stickyMovement)
-			{
-				item.TryTilePush(PushDirectionPosition.RoundTo2Int(), null , ConveyorBeltSpeed);
-			}
-			else
-			{
-				item.NewtonianPush(PushDirectionPosition.RoundTo2Int(), ConveyorBeltSpeed - item.NewtonianMovement.magnitude, 0.5f);
-			}
-
+			item.TryTilePush(PushDirectionPosition.RoundTo2Int(), null , ConveyorBeltSpeed);
```

Comment:
still has the same issue, because momentum is heritage from a tile push for items and objects, basically non-sticky movement stuff, Though I suppose it can go round corners better

Reply:
So far I haven't encountered any of the issues that I intended to fix; if there are other stuff that needs to be fixed, then it can wait for another PR.

Reply:
How jerky does the movement look? I suppose it's getting merged away but I just wondered how it looks